### PR TITLE
DIABLO-474, restore kaltura events.to_api_json with option for raw-resultset

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/DIABLO-474

### CODE REVIEW

File: diablo/externals/kaltura.py [hunk 1]
```diff
@@ -193,7 +193,7 @@ def schedule_recording(
             term_id=term_id,
         )
         recurrences_filter = KalturaScheduleEventFilter(parentIdEqual=kaltura_schedule.id)
-        for recurrence in self._get_events(kaltura_event_filter=recurrences_filter):
+        for recurrence in self._get_events(kaltura_event_filter=recurrences_filter, organize_results=False):
```
Comment (johncrossman) [inline diablo/externals/kaltura.py:196]: We're querying for recurrences of the Kaltura schedule and then cancelling instances that have blackout-date conflicts. In this case, we want the raw list of event_json objects.

File: diablo/externals/kaltura.py [hunk 2]
```diff
@@ -503,7 +503,16 @@ def _event_to_json(event):
                 }
             return api_json
 
-    return [_event_to_json(event) for event in scheduled_events]
+    raw_list = [_event_to_json(event) for event in scheduled_events]
+    if organize_results:
+        recurring_events = list(filter(lambda e: e['recurrenceType'] == 'Recurring', raw_list))
+        standalone_events = list(filter(lambda e: e['recurrenceType'] not in ['Recurrence', 'Recurring'], raw_list))
```
Comment (paulkerschen) [inline diablo/externals/kaltura.py:509]: @johncrossman Is this right? Recurring events have type 'Recurring', and standalone events are all other events except those with type 'Recurrence,' which we ignore?

Comment (johncrossman) [reply diablo/externals/kaltura.py:509]: @paulkerschen I reverted to what was there before... And what was there before was composed when I was new to the Kaltura API.  I'll have a closer look tomorrow and decide if improvements are needed. Thanks for asking!

**Comment by johncrossman** [comment]
@paulkerschen - PR amended. Nothing will be lost during `_events_to_api_json()`.  Recurring events still get their rightful recurrences (children); otherwise, events are included as independents. For example, when we query for... 
- events in a room -> we get a hodge-podge
- events in a series -> we get only 'recurrences'
- blackout events -> we get neither recurring nor recurrence

`?w=1` will give you a much cleaner diff. Thanks!

**Comment by paulkerschen** [comment]
Makes sense, or as much sense as this API makes. Thanks!

**Pull request merged**